src/components/sections/getstarted.js

```diff
@@ -60,7 +56,13 @@ const TryItButton = styled.button`
   border-color: initial;
   border-image: initial;
   outline: 0px;
+  display: inline-flex;
+  align-items: center;
+  text-decoration: none;
+  transition: all 0.2s ease;
   &:hover {
+    transform: translateY(-1px);
+    background: ${props => props.theme.color.accent};
```

Comment:
Were you proposing that a new color value be added to the `theme` file for the `hover` state that has the same value as `accent`? If so, that sounds good to me as in the future the `accent` color and `hover` color may diverge.

Reply:
Exactly, great I'll update the PR